packages/mobile/src/escrow/ReclaimPaymentConfirmationScreen.tsx

```diff
@@ -97,6 +98,13 @@ class ReclaimPaymentConfirmationScreen extends React.Component<Props> {
   render() {
     const { t, fee } = this.props
     const payment = this.getReclaimPaymentInput()
+    const convertedAmount = divideByWei(payment.amount.toString())
+    if (!fee) {
+      Logger.error(TAG, 'Error fetching suggested reclaim fee, show error message')
```

Comment:
Render functions should be pure (no side-effects). 
We need to remove this logger call and error dispatch. Why is this here? Can the sagas that set fee handle this error setting? And what should this render if fee isn't set?